src/chains/ethereum/ethereum/tests/transaction-pool.test.ts

```diff
@@ -95,421 +96,714 @@ describe("transaction pool", async () => {
         latest: { header: { baseFeePerGas: Quantity.from(875000000) } }
       }
     };
-  });
-  beforeEach(async function () {
-    // for each test, we'll need a fresh set of origins
+
     origins = new Map();
-  });
+  };
 
-  it("rejects transactions whose gasLimit is greater than the block gas limit", async () => {
```

Comment:
This big block is just rearranging to prevent subsequent tests from relying on previous ones being run (`.only` was broken before this)